### PR TITLE
Update UG_get

### PR DESCRIPTION
**Pull request by hqhqhq1**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -165,9 +165,23 @@ Example:
 
 ### Getting: `get`
 
-#### Getting …:
+#### Getting a Module details: `get -m`
 
-#### Getting …:
+Format: `get -m MODULE…`
```
Comment (LowJiaHao99) [inline docs/UserGuide.md:170]: image

Is this command used to get details of a module or details of multiple modules?
If it is the latter, line 168 can be misleading
If it is the former, `...` should not be included in line 170

File: docs/UserGuide.md [hunk 2]
```diff
@@ -165,9 +165,23 @@ Example:
...
+
+* Gets all the classes and students in the given module
+
+Example:
+* `get -m CS2103T` to view all the classes and students added to CS2103T
```
Comment (LowJiaHao99) [inline docs/UserGuide.md:175]: should it be added to the module `CS2103T` or added to CS2103T?

This is to standardize, the language of line 175 and line 184

File: docs/UserGuide.md [hunk 3]
```diff
@@ -165,9 +165,23 @@ Example:
...
+
+#### Getting a Class details: `get -c`
+
+Format: `get -c MODULE CLASS`
+
+* Gets the specified class based on the given module
+
+Example:
+* `get -c CS2103T G08` view the class `G08` from the module `CS2103T`
```
Comment (LowJiaHao99) [inline docs/UserGuide.md:184]: ... view the all the students in the class `G08`

would this be clearer?

This is to standardize, the language of line 175 and line 184

**Review by LowJiaHao99 — COMMENTED** [review]
Minor issues, once resolved will be LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (hqhqhq1) [reply docs/UserGuide.md]: get detail for a module so ... is removed

File: docs/UserGuide.md (hunk 2 above)
Comment (hqhqhq1) [reply docs/UserGuide.md]: okay!

File: docs/UserGuide.md (hunk 3 above)
Comment (hqhqhq1) [reply docs/UserGuide.md]: done!

**Comment by LowJiaHao99** [comment]
LGTM

**Pull request merged**